content/docs/rfcs/52/README.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: 52
+title: 52/WAKU2-RELAY-STATIC-SHARD-ALLOC
+name: Waku v2 Relay Static Shard Allocation
+status: raw
+category: Informational
+tags:
+editor: Daniel Kaiser <danielkaiser@status.im>
+contributors:
+---
+
+# Abstract
+
+This document lists static shard flag index assignments (see [51/WAKU2-RELAY-SHARDING](/spec/51/).
+
+# Background
+
+Similar to the [IANA port allocation](https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml),
+this document lists static shard index assignments (see [51/WAKU2-RELAY-SHARDING](/spec/51/).
+
+The index ranges are as follows:
+index `0` represents the global shards,
+indices `1` to `15` are reserved,
+indices `16` to `1023` are reservable for apps,
+indices `1024` to `49151` are for open use by apps,
+indices `49152` to `65535` are reserved.
+
+# Assingment Process
+
+> *Note*: Future versions of this document will specify the assignment process.
+
+# List of Static Shard Indices
+
+|    index | Protocol/App | Description                         |
+|    ---   | ---          |                                     |
+|      0   |   global     | global use                          |
+|      1   |   reserved   |                                     |
+|      2   |   reserved   |                                     |
+|      3   |   reserved   |                                     |
+|      4   |   reserved   |                                     |
+|      5   |   reserved   |                                     |
+|      6   |   reserved   |                                     |
+|      7   |   reserved   |                                     |
+|      8   |   reserved   |                                     |
+|      9   |   reserved   |                                     |
+|     10   |   reserved   |                                     |
+|     11   |   reserved   |                                     |
+|     12   |   reserved   |                                     |
+|     13   |   reserved   |                                     |
+|     14   |   reserved   |                                     |
+|     15   |   reserved   |                                     |
+|     16   |   Status     |  Status main net                    |
```

Comment:
Since waku is permissionless how do you enforce this? I mean, this could be an internal recommendation but any app can send messages to Status mainnet shard. So wondering about the impact it will have if people don't respect this.